src/commands/test.js

```diff
@@ -0,0 +1,498 @@
+const chalk = require('chalk')
+const compose = require('docker-compose')
+const fs = require('fs-extra')
+const http = require('http')
+const net = require('net')
+const tmp = require('tmp')
+const Docker = require('dockerode')
+const path = require('path')
+const stripAnsi = require('strip-ansi')
+const util = require('util')
+const exec = util.promisify(require('child_process').exec)
+const spawn = require('child_process').spawn
+
+const { createCompiler } = require('../command-helpers/compiler')
+const { fixParameters } = require('../command-helpers/gluegun')
+const { step, withSpinner } = require('../command-helpers/spinner')
+
+// Clean up temporary files even when an uncaught exception occurs
+tmp.setGracefulCleanup()
+
+const HELP = `
+${chalk.bold('graph test')} [options] ${chalk.bold('<test-command>')}
+
+Options:
+
+  -h, --help                    Show usage information
+      --node-logs               Print the Graph Node logs (optional)
+      --compose-file <file>     Custom Docker Compose file for additional services (optional)
+      --node-image <image>      Custom Graph Node image to test against (default: graphprotocol/graph-node:latest)
+      --standalone-node <cmd>   Use a standalone Graph Node outside Docker Compose (optional)
+      --standalone-node-args    Custom arguments to be passed to the standalone Graph Node (optional)
+      --skip-wait-for-ipfs      Don't wait for IPFS to be up at localhost:5001 (optional)
+      --skip-wait-for-ethereum  Don't wait for Ethereum to be up at localhost:8545 (optional)
+      --skip-wait-for-postgres  Don't wait for Postgres to be up at localhost:5432 (optional)
+`
+
+module.exports = {
+  description: 'Tests the current project against a Graph Node and Parity testnet',
```

Comment:
Do you mean ganache testnet?

Reply:
I initially tried Parity, but I should update the description anyway.

Reply:
Reworded